app/assets/stylesheets/lite_mode.scss

```diff
@@ -0,0 +1,156 @@
+body {
+  font-family: system, -apple-system, BlinkMacSystemFont, ‘Segoe UI’, Roboto, Oxygen, Ubuntu, Cantarell, ‘Open Sans’, ‘Helvetica Neue’, sans-serif;
+
+  img {
+    display: none;
+  }
+
+  &#products {
+    .product {
+      margin-top: 1rem;
+      border-top: 1px solid #ddd;
+      padding-top: 1rem;
+
+      .product-front {
+        > a {
+          display: none;
+        }
+      }
+
+      .product-main {
+        > header {
+          display: none;
```

Comment:
Why is this being hidden? It seems important.

For example, this zine page:

`https://crimethinc-staging-pr-912.herokuapp.com/zines/secret-world-of-duvbo`

It doesn't have a title/subtitle and the nav crumb trail. 

Compared to:

`https://crimethinc.com/zines/secret-world-of-duvbo`

Reply:
I hid that because if not, the titles are doubled up on the books page. I will try and do some more specific targeting for the books page. No zines show up on the Zine page in my dev env so I didn't see that the classes were reused on those.